.github/unittest.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
```

Comment:
Although it seems weird to have a standalone script for a single workflow, this script works fine for Linux as well. See #7149. I'll migrate them to this script next.

Reply:
See #7189.